tools/codemods/CONTRIBUTING.md

```diff
@@ -0,0 +1,150 @@
+# Contributing to codemods
+
+## Getting Started
+
+All codemods/migrations can be found under `src/migrations` and each migration should have its own directory.
+
+```
+src
+ ┣ migrations                                     # directory for all migrations
+ ┃ ┣ <name-of-migration>                          # directory for individual migration
+ ┃ ┃ ┣ tests                                      # directory for migration tests
+ ┃ ┃ ┃ ┗ <name-of-migration>                      # directory for a specific test
+ ┃ ┃ ┃ ┃ ┣ <name-of-migration>.input.tsx          # input file for test
+ ┃ ┃ ┃ ┃ ┣ <name-of-migration>.output.tsx         # output file for test
+ ┃ ┃ ┃ ┃ ┗ transform.spec.ts                      # jest test file
```

Comment:
are there too many nesting levels here?

Reply:
i think that i can get rid of the directories for a specific test, Ln12